target/linux/ipq807x/image/generic.mk

```diff
@@ -23,6 +23,12 @@ define Device/UbiFit
 	IMAGE/nand-sysupgrade.bin := sysupgrade-tar | append-metadata
 endef
 
+define Device/QnapImage
```

Comment:
Is there a generic recipe for these, they don't really look special to me at all

Reply:
Not as I know it.  I find this incl. the padding only device specific in other targets.
Should I remove the image recipe and move the image generation to the device/qnap_301w?

Reply:
I've removed the QnapImage recipe and added the image definition to the device itself. I hope it's allright.

Reply:
That is fine as there is only one device using it, if there are multiple then it makes sense to just reference the recipe